docs/tutorial.md

```diff
@@ -849,7 +849,8 @@ the Table module:
 ```
 
 We can then manipulate the dataframe as we see fit, e.g. by saving it
-as a csv file:
+as a csv file (note that it can be different from the precalculated ones in 
+the tutorial due to database updates):
```

Comment:
this is fine, but you also need to work on the COSMIC section of this file to update the function calls and check if we say re supporting alternative isoforms